src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/SmallFullService.java

```diff
@@ -14,6 +16,15 @@
      */
     void createSmallFullAccounts(String transactionId, String companyAccountsId) throws ServiceException;
 
+    /**
+     * Update a small full resource
+     * @param periodEndOn The updated period end date you want to submit
```

Comment:
Maybe adapt this to include the `null` condition? Something like "The period end date for which to update the resource. If `null` is provided, the date will be defaulted to that of the company" or similar?